src/main/resources/application.conf

```diff
@@ -1,32 +1,40 @@
-etc-client {
+grothendieck {
+  # Identifier used when connecting to other clients
+  client-id = "grothendieck"
 
-  client-id = "etc-client"
+  # Version string (reported by an RPC method)
+  client-version = "grothendieck/v0.1"
 
-  client-version = "etc-client/v0.1"
+  # Base directory where all the data used by the node is stored, including blockchain data and private keys
+  datadir = "~/.grothendieck/"
 
-  datadir = "/tmp/etc-client/"
+  # The unencrypted private key of this node
+  keys-file = ${grothendieck.datadir}"nodeId.keys"
```

Comment:
Agreed. But I still think it would be even better to use [relative vs absolute](https://github.com/input-output-hk/etc-client/pull/246#discussion_r126969209) paths